src/main/msp/msp.c

```diff
@@ -1225,6 +1225,9 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
         sbufWriteU16(dst, currentControlRateProfile->rate_limit[FD_PITCH]);
         sbufWriteU16(dst, currentControlRateProfile->rate_limit[FD_YAW]);
 
+        // added in TODO
+        sbufWriteU8(dst, currentControlRateProfile->throttle_vbat_limit_percent);
```

Comment:
This can not be added now - the approach in Betaflight is to wait for one release before adding new parameters to MSP to ensure they are stable.